src/utils.py

```diff
@@ -4964,7 +4964,9 @@ def get_label_pno(pgNo, labels):
     rule = rule_dict(item)
     prefix = rule.get("prefix", "")
     style = rule.get("style", "")
-    pagenumber = pgNo - rule["startpage"] + rule["firstpagenum"]
+    # make sure we start at 0 when enumerating the alphabet
+    delta = -1 if style in ("a", "A") else 0
+    pagenumber = pgNo - rule["startpage"] + rule["firstpagenum"] + delta
```

Comment:
Could we instead change `rule_dict()` to either add one to `rule["startpage"]` or subtract one from `rule["firstpagenum"]`, if it finds a style specification that is 'a' or 'A'? Or would that be doing the wrong thing?

Reply:
I thought about this too, but that would require some other code to be changed. This in the end was the simplest thing to do.
Ultimately I want to replace all this with the native MuPDF page label support, so it simply is not worth investing a lot of thought here.